### PR TITLE
add test domains

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:
- add test domains. 

Could this have been done with loops? probably. I didn't use loops because:
- I don't fully grok how to do that in terraform
- this isn't actually a dynamic list - it's a static list that happens to follow a pattern

## security considerations
None

### CODE REVIEW

File: terraform/modules/external_domain_broker_tests/resources.tf [hunk 1]
```diff
@@ -19,12 +19,105 @@ resource "aws_route53_record" "record" {
...
+  name = "cloud-gov-test-domain-0.com"
+}
+resource "aws_route53_record" "record" {
+  name    = "${aws_route53_zone.stack_test_0.name}"
+  zone_id = "${data.aws_route53_zone.test_0.zone_id}"
+  type    = "NS"
+  ttl     = "60"
+
+  records = [
+    "${aws_route53_zone.test_0.name_servers.0}",
```
Comment (soutenniza) [inline terraform/modules/external_domain_broker_tests/resources.tf:38]: Is this pointing to line 23?

Comment (bengerman13) [reply terraform/modules/external_domain_broker_tests/resources.tf:38]: :doh:

Comment (spgreenberg) [reply terraform/modules/external_domain_broker_tests/resources.tf:38]: It is pointing to line 28, no?

Comment (bengerman13) [reply terraform/modules/external_domain_broker_tests/resources.tf]: yeah, it's pointing to 28 but it's supposed to be pointing to 23
updated in d04bcdf10c30de926cd60969501b810072fe4ece

File: terraform/modules/external_domain_broker_tests/resources.tf [hunk 2]
```diff
@@ -19,12 +19,105 @@ resource "aws_route53_record" "record" {
...
 data "template_file" "policy" {
   template = "${file("${path.module}/policy.json")}"
 
   vars {
     aws_partition = "${var.aws_partition}"
-    hosted_zone = "${aws_route53_zone.zone.zone_id}"
+    hosted_zone_0 = "${aws_route53_zone.zone.zone_id}"
+    hosted_zone_1 = "${aws_route53_zone.test_0.zone_id}"
+    hosted_zone_2 = "${aws_route53_zone.zone_1.zone_id}"
+    hosted_zone_3 = "${aws_route53_zone.zone_2.zone_id}"
```
Comment (soutenniza) [inline terraform/modules/external_domain_broker_tests/resources.tf:119]: are these supposed to be test_1 instead of zone_1?

Comment (bengerman13) [reply terraform/modules/external_domain_broker_tests/resources.tf:119]: yes.

Comment (bengerman13) [reply terraform/modules/external_domain_broker_tests/resources.tf]: updated in 00bde208e10ba3f62629cdde8a7fbe73010745c0

**Pull request merged**